src/stories/AreaChart.stories.tsx

```diff
@@ -53,14 +54,24 @@ export const Default: Story = {
         <YAxis {...defaultYAxisProps} dataKey="y" />
         <XAxis {...defaultXAxisProps} dataKey="x" />
         <ReferenceLine {...defaultReferenceLineProps} x="Wed" />
-        <ChartTooltip 
-            content={({ active, payload, label }) => <DefaultTooltip label={label} active={active} payload={payload} valueFormatter={(p)=> `${p.y} count`} />}
+        <ChartTooltip
+          {...defaultTooltipProps}
```

Comment:
{...defaultTooltipProps} is a breaking change.

Three possible ways forward:
a) keep the breaking change, be sure to update it in the client
b) make an exception for `ChartTooltip` to use extended class with defaultProps.
c) try to refactor everything to class extensions with default props

i'd go with a)

Reply:
We can keep the breaking change it should be cheap to fix!